src/main/java/ch/epfl/biop/imaris/demo/MakeAndGetSpotsDemo.java

```diff
@@ -7,13 +7,13 @@
 
 /**
  * EasyXT Demo
- *
+ * <p>
```

Comment:
what does this <p> do here? What is the use of having it?

Reply:
code > Reformat Code add them

Reply:
f0f6e7d30135049c66e7173415432a2f3ba4922f

Reply:
Huh, I'll have a look. We can leave it for now